util/chplenv/utils.py

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env python
```

Comment:
utils.py doesn't have a main function and is only a helper module used by other chplenv scripts. I think we should remove it's execute permission, instead of adding the shebang